src/Components/Movies.js

```diff
@@ -0,0 +1,50 @@
+import axios from "axios";
+import { Component } from "react";
+
+class Movies extends Component {
+    state = {movie:[], selectedMovieTitle: "", selectedMovie: {} }
+  
+    fetchAllMovies = async () => {
+    try {
+      const res = await axios.get("https://ghibliapi.herokuapp.com/films");
+      this.setState({movie: res.data})
+    } catch (error) {
+        this.setState({movie: []})
+    }
+  };
```

Comment:
I would work on how your code lines up. What's inside your function (including the try/catch should be indented and the curly brace should like up with the function. This makes it look like you're closing the component